android_docker_image.md

```diff
@@ -0,0 +1,109 @@
+The semaphore `android` docker image is a customized image based on `ubuntu:16.04`
+optimized for CI/CD. It comes with a set of preinstalled utility tools commonly
+used for CI/CD workflows.
+
+[[__TOC__]]
+
+## Using the semaphoreci/android docker image in your semaphore.yml configuration
+
+To use the images in this repository for your CI/CD pipelines, set up your Semaphore YAML with:
```

Comment:
This should explain for what projects this configuration is good for. Kotlin, XYZ...

Reply:
This example should be followed by an example for React Native. That should be enough info for 90+% of people.